AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.java

```diff
@@ -680,7 +680,7 @@ public boolean onCreateOptionsMenu(Menu menu) {
             suspend_icon.setTitle(R.string.menu_suspend_card);
         }
         if (buryNoteAvailable()) {
-            bury_icon.setIcon(R.drawable.ic_flip_to_back_white_24px_dropdown); // FIXME NEEDS A "DROPDOWN" VERSION WITH BOTTOM RIGHT TRIANGLE (AND PURGE EXISTING)
+            bury_icon.setIcon(R.drawable.ic_flip_to_back_dropdown); // FIXME NEEDS A "DROPDOWN" VERSION WITH BOTTOM RIGHT TRIANGLE (AND PURGE EXISTING)
```

Comment:
Stale comment now - you have fixed the fixme :+1: 

```suggestion
            bury_icon.setIcon(R.drawable.ic_flip_to_back_dropdown);
```